### DISCUSSION TITLE
Audio that plays automatically?

### DISCUSSION THREAD

**Discussion by MusicMasterrMatt** [mozilla/hubs#4193] [General Questions & Discussion] [answered]
Hello, is there a way to have audio play automatically for a more immersive experience and have different audio at certain times on or near certain things?

**Comment by johnshaughnessy** [comment, marked as answer]
Partially yes. You can add audio/video nodes to a scene in Spoke and configure the panner node settings to determine falloff range. 

![image](https://user-images.githubusercontent.com/4072106/116120385-548ee480-a674-11eb-833b-7dac2066869a.png)
![image](https://user-images.githubusercontent.com/4072106/116120436-607aa680-a674-11eb-87cc-0a044dfddc27.png)



But there is no way to dynamically trigger audio at a given time.

**Reply by MusicMasterrMatt** [reply]
Brilliant thank you however is there a way to add our own audio? that is not visible to the audience? 

regards,

Matt

**Comment by dezigns333** [comment]
Audio URL

**Reply by MusicMasterrMatt** [reply]
Thank you very much
